acceptance/tests/api-gateway_v2/api_gateway_v2_test.go

```diff
@@ -159,7 +159,7 @@ func TestAPIGateway_V2_Basic(t *testing.T) {
 
 			// check our finalizers
 			require.Len(t, gatewayClass.Finalizers, 1)
-			require.EqualValues(t, gatewayClassFinalizer, gatewayClass.Finalizers()[0])
```

Comment:
This test is unused, but the fix is required to satisfy the linter after the `control-plane` version bump.

Reply:
cc @nathancoleman @sarahalsmiller since I'm not certain this is exactly the end product you want for this test. All this is doing is referencing the finalizers `[]string` directly since the accessor was removed in the `proto-public` migration.